index.bs

```diff
@@ -904,21 +906,30 @@ of a given agent's behavior over time.
 Delegation {#delegation}
 ----------
 
-Client Hints will be delegated from top-level pages via Permissions Policy. This reduces the likelihood that [=user agent=]
-information will be delivered along with subresource requests, which reduces the potential for
+Client Hints will be delegated from top-level pages via Permissions Policy
+([[!permissions-policy-1]]). This reduces the likelihood that [=user agent=] information will be
+delivered along with subresource requests, which reduces the potential for
 [=passive fingerprinting=].
 
 That delegation is defined as part of [=append client hints to request=].
 
 Fingerprinting {#fingerprinting}
 --------------
 
-The primary goal of User Agent Client Hints is to reduce the default entropy
-available to a server for [=passive fingerprinting=]. However, it will still be possible
-for some, or all, hints to be requested and used for [=active fingerprinting=] purposes by
-first or delegated third parties. As noted in [[#access]], [=User agents=] should consider
-policies to restrict or reduce access to parties that are known to <a lt="active fingerprinting">
-actively fingerprint</a> their users.
+The *primary goal* of User Agent Client Hints is to reduce the amount of default entropy exposed to
+the web at large through the User-Agent header field, which may be used for
+[=passive fingerprinting=] purposes.
+
+[=User agents=] naturally vary in the types and values of information exposed by the UA
+string and some have made deliberate privacy-focused choices to not expose certain bits, such
+as device [=model=] on mobile. This specification makes no requirements for a [=user agent=] to
+expose more entropy via User Agent Client Hints than it does through the User-Agent header field
+today—the empty string is a valid value for all such high-entropy hints.
+
+However, it will still be possible for some, or all, hints to be requested and used for
+[=active fingerprinting=] purposes by first or delegated third parties. As noted in [[#access]],
+[=user agents=] should consider policies to restrict or reduce access to parties that are known to
+<a lt="active fingerprinting">actively fingerprint</a> their users.
```

Comment:
I'm also seeing a need to strengthen the access restrictions section cited here.  It doesn't concretely say what might happen if a browser decides to restrict access.

Reply:
Thanks, agree it can be improved. I've filed https://github.com/WICG/ua-client-hints/issues/266 to tackle this.